### PR TITLE
Created a survey result page that doesn't require password

### LINKED ISSUE #49: Create survey result page but without password
This shows result to the public without requiring a password

### PR DESCRIPTION
**Pull request by loisaleghe**
This is an optional feature but if the user doesn't mind the public seeing the results they can choose this option if they do they can say no too.

### CODE REVIEW

**Comment by loisaleghe** [comment]
test has not been added yet

**Comment by JulianMendoza** [comment]
You could probably just reuse the existing code to display the results with a toggle variable but if this works its okay! Please add tests to this pull request, it's part of the CI/CD pipeline and it will definitely be something that the TA is looking for. Other than that, looks great!

**Comment by loisaleghe** [comment]
> You could probably just reuse the existing code to display the results with a toggle variable but if this works its okay! Please add tests to this pull request, it's part of the CI/CD pipeline and it will definitely be something that the TA is looking for. Other than that, looks great!

okay, I will do that right away

**Comment by loisaleghe** [comment]
> You could probably just reuse the existing code to display the results with a toggle variable but if this works its okay! Please add tests to this pull request, it's part of the CI/CD pipeline and it will definitely be something that the TA is looking for. Other than that, looks great!

Also it is true that I could have used existing code already present but since the existing required both title and password to be true I wanted to create one without password that would be separate in case the existing one breaks it doesn't fully affect this , it also felt more straight forward that way but for future development to reduce stress I'll probably do what you suggested!!

**Comment by loisaleghe** [comment]
the test has been added now

**Review by JulianMendoza — APPROVED** [review]

File: src/main/java/com/surveyMonkey/controllers/AdminController.java [hunk 1]
```diff
@@ -135,4 +155,21 @@ public String deleteSurvey(@RequestParam("surveyCode") String surveyCode, @Reque
         }
         return "deleteSurvey";
     }
+    @PostMapping({"/deleteSurveyWithoutPswrd"})
```
Comment (oyenmwen) [inline src/main/java/com/surveyMonkey/controllers/AdminController.java]: What is the purpose of delete survey without a password feature? Shouldn't be an admin/authorized only capability ? Shouldnt only the owner be allowed to delete the survey ?

**Review by oyenmwen — CHANGES_REQUESTED** [review]

File: src/main/java/com/surveyMonkey/controllers/AdminController.java (hunk 1 above)
Comment (loisaleghe) [reply src/main/java/com/surveyMonkey/controllers/AdminController.java]: > What is the purpose of delete survey without a password feature? Shouldn't be an admin/authorized only capability ? Shouldnt only the owner be allowed to delete the survey ?

You are right only the owner should be able to delete survey, I didn't think of that when I was writing the code, good catch, thank you!

**Comment by loisaleghe** [comment]
I have fixed it now

**Review by oyenmwen — APPROVED** [review]
Looks great Wendy good job

**Pull request merged**